dss/storage/validator.py

```diff
@@ -1,23 +1,91 @@
 """Utilities in this file are used to removed extra fields from index data before adding to the index."""
+import io
 import re
+
+import boto3
+from cloud_blobstore import BlobNotFoundError
 from jsonschema import validators
 from jsonschema import _utils, _validators
 
 import json
-from urllib.request import urlopen
-from urllib.error import URLError
-import ssl
-from typing import List
+from typing import List, Optional
 import logging
 
+from requests import RequestException
+
+from dss import Replica, Config, DSSException
+from hashlib import sha1
+import requests
+
+
+DSS_Draft4Validator = validators.create(meta_schema=_utils.load_schema("draft4"),
+                                        validators={u"$ref": _validators.ref,
+                                                    u"additionalProperties": _validators.additionalProperties,
+                                                    u"properties": _validators.properties_draft4,
+                                                    u"required": _validators.required_draft4
+                                                    },
+                                        version="draft4"
+                                        )
```

Comment:
Why is "draft4" (or any other metadata schema version) hardcoded in our code? It should be parameterized. (I realize this was added earlier - sorry I missed the earlier review)

Reply:
That's draft4 of the JSON schema spec, not the HCA metadata schema.

Reply:
My bad, got it.